community/kernel/src/main/java/org/neo4j/kernel/impl/store/DumpStore.java

```diff
@@ -55,7 +55,7 @@ public static void main( String... args ) throws Exception
         try ( PageCache pageCache = createPageCache( fs ) )
         {
             DefaultIdGeneratorFactory idGeneratorFactory = new DefaultIdGeneratorFactory();
-            StoreFactory storeFactory = new StoreFactory( new Config(), idGeneratorFactory, pageCache, fs, logProvider(), null );
+            StoreFactory storeFactory = new StoreFactory( null, new Config(), idGeneratorFactory, pageCache, fs, logProvider(), null );
```

Comment:
Apparently so! If the config didn't contain `store_dir` previously, then it would have ended up as `null` inside the factory. As you can see, it was previously `new Config()`.

That's not to say it's correct. It's almost certainly not.